eve/methods/common.py

```diff
@@ -1047,3 +1047,23 @@ def oplog_push(resource, document, op, id=None):
 
     if entries:
         app.data.insert(config.OPLOG_NAME, entries)
+
+    def _client_projection(self, req):
```

Comment:
hum, what part?

My plan is to adjust this function to be able to deal with different level of projection, like:
```projection={"my_embedded_resource.key":1}.```
I may be wrong, but I don't think this is something that work for the moment.